client/app/visualizations/chart/plotly.js

```diff
@@ -252,7 +252,9 @@ const PlotlyChart = () => {
           const cellWidth = 1 / cellsInRow;
           const cellHeight = 1 / rows;
           const xPadding = 0.02;
-          const yPadding = 0.05;
+          const yPadding = 0.1;
```

Comment:
Why did we need to change this?

Reply:
We need a little more space than now to make the second-row title to fit. The gap between the first row and second row is too tight for a row of text to fit nicely.